packages/block-library/src/site-logo/edit.js

```diff
@@ -674,8 +673,7 @@ export default function LogoEdit( {
 					mediaLibraryButton={ ( { open } ) => {
 						return (
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen, while you add `core/site-logo` block, check for upload button.

**Note:** This has some styles that have higher specificity and specific to this upload button on this block, hence before and after is same, but we can see our change is applied in inspector controls.

| Before | After     |
| :-------- | :------- |
| <img width="1331" alt="site-logo-2-before" src="https://github.com/user-attachments/assets/5254a899-a0ea-4d4b-9942-be531b9a0060"> | <img width="1331" alt="site-logo-2-after" src="https://github.com/user-attachments/assets/3fa6bbf1-021e-488a-abac-0018b883c120"> |

Reply:
Same as above. There are again a lot of [style overrides](https://github.com/WordPress/gutenberg/blob/ff01bcbd4aeb84d6a28a655ca652a9ad9f16cebe/packages/block-library/src/site-logo/editor.scss), which ideally shouldn't be there at all.

The code change here is good for the scope of this PR